tests/Datagrid/PagerTest.php

```diff
@@ -29,7 +28,6 @@ public function testComputeNbResultFoCompositeId(): void
     {
         $em = DoctrineTestHelper::createTestEntityManager();
         $classes = [
-            $em->getClassMetadata(User::class),
```

Comment:
There is a typo in the test name, and the dataProvider seems useless.

I don't know what the test was testing, but since you've to modify it in order to really test the Composite case, I feel like it was testing the single case.
Since it's modified, we don't have tests for single case anymore. Can u add one ?

Reply:
i restore test single identifier